### DISCUSSION TITLE
Having trouble installing Ruby with Homebrew.

### DISCUSSION THREAD

**Discussion by yiani** [Homebrew/discussions#1470] [Getting started]
I am trying to install Ruby through Homebrew but this keeps coming up and I do not understand how to fix or solve it. 

Warning: Calling `sha256 "digest" => :tag` in a bottle block is deprecated! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
Please report this issue to the homebrew/core tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/readline.rb:14

any help is much appreciated.

**Comment by SMillerDev** [comment]
What does `brew config` say?

**Reply by yiani** [reply]
yianiMacBookPro:~ ioanniskapetanakis$ brew config
HOMEBREW_VERSION: 3.1.7-38-g496c24a
ORIGIN: https://github.com/Homebrew/brew
HEAD: 496c24aae223359084907fb9c665dec3d487cf71
Last commit: 7 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 82ae7037bd677d2591bb99a7103f7c3da90a7ece
Core tap last commit: 1 year, 11 months ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 2
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: dual-core 64-bit penryn
Clang: 8.0.0 build 800
Git: 2.10.1 => /usr/local/bin/git
Curl: 7.43.0 => /usr/bin/curl
macOS: 10.11.6-x86_64
CLT: 8.2.0.0.1.1480973914
Xcode: N/A

**Reply by gromgit** [reply]
Your Homebrew software is up to date, but the core formula tap is almost _2 years old_! That's the primary source of your issue: The general formula syntax has evolved significantly during those 2 years.

Since El Capitan hasn't been supported in a while, you should probably run `brew doctor` and see if there's anything that needs fixing first, before running `brew update-reset` to force everything back in sync.

**Reply by yiani** [reply]
Thanks for all of your help! Everything is solved. My end goal solution was to get Jekyll up and running to learn to build static websites. Everything is working well on this old 2009 MBP!! Thanks again for all of your help, I really appreciate it!!
